templates/cmd/controller/main.go.tpl

```diff
@@ -62,7 +62,13 @@ func main() {
 	flag.Parse()
 	ackCfg.SetupLogger()
 
-	if err := ackCfg.Validate(); err != nil {
+	managerFactories := svcresource.GetManagerFactories()
+	resourceKinds := make([]string, 0, len(managerFactories))
+	for _, mf := range managerFactories {
```

Comment:
Maybe we should just pass the entire `GroupKind`, so that different downstream validation can use whichever naming and casing it needs

Reply:
Agreed. Changed to `GroupKinds`